ClassMoodApp/templates/createPollingQuestion.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layout.html" %}
+{% block body %}
+<nav class="navbar navbar-inverse navbar-top">
```

Comment:
All new html pages will be extending from baseHomePage.html which all ready contains the navbar. This is a page I created, so after merging, ensure that your html is extending from that page, and change the block name to newPage.